app/views/conference/_schedule.html.erb

```diff
@@ -55,6 +55,12 @@ $(function() {
     }
 });
 
+// Open current day's schedule by default
+$(function() {
+    today = new Date().toISOString().split('T')[0];
+    $('.date-tab[href="#' + today +'"]').trigger( "click" );
+});
+
```

Comment:
This doesn't take the conference time zone into account so I doubt that it works properly. 

I was already working on that as I was working in the public schedule already, I have just realised that you too. :confounded:

I have just added a commit to https://github.com/openSUSE/osem/pull/1032 in which the schedule is open in the current day and **time**, taking into account the **time zone**.